stable/artifactory-ha/values.yaml

```diff
@@ -1570,3 +1570,370 @@ filebeat:
     output:
       logstash:
          hosts: ["{{ .Values.filebeat.logstashUrl }}"]
+
+## Prometheus Exporter / Metrics
+##
+metrics:
+  enabled: false
+
+  service:
+    port: 24231
+    type: ClusterIP
+
+  ## Prometheus Operator ServiceMonitor configuration
+  ##
+  serviceMonitor:
+    labels: {}
+
+    ## Interval at which metrics should be scraped
+    ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#endpoint
+    ##
+    interval: 15s
+
+    ## Timeout after which the scrape is ended
+    ## ref: https://github.com/prometheus-operator/prometheus-operator/blob/master/Documentation/api.md#endpoint
+    ##
+    scrapeTimeout: 15s
```

Comment:
Should `scrapeTimeout` and `interval` really have the same value? Sounds like it might be problematic.

Reply:
My thinking was that this would limit any individual scrape to the interval between them.  Which one would you suggest changing?

Reply:
I think we can just leave it empty. WDYT? According to the docs I found, this is not mandatory. Just wrap each individual value with a `with` statement when you use it and it will act as an `if` statement

Reply:
That makes sense, leave it up to the consumers of the chart.